cypress/integration/ui/dashboard_filter/dashboard_filter.js

```diff
@@ -0,0 +1,26 @@
+import { When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+const OPTIONS = { timeout: 15000 }
+
+When('I add a {string} filter', () => {
+    cy.contains('Add filter').click()
+
+    cy.get('[data-test="undefined-button-pe"]').click()
```

Comment:
`undefined-button-pe`, why the `undefined`?

Reply:
That's coming from Analytics. But not using this selector anymore.